modules/plymouth/nixos.nix

```
@@ -26,19 +26,9 @@ let
       else "cp ${./theme_still.script} $themeDir/stylix.script"
     }
 
-    ${
-      if cfg.blackBackground
-      then ''
-        substituteInPlace $themeDir/stylix.script \
-          --replace "%BASE00%" "0, 0, 0" \
-          --replace "%BASE05%" "1, 1, 1"
-      ''
-      else ''
-        substituteInPlace $themeDir/stylix.script \
-          --replace "%BASE00%" "${base00-dec-r}, ${base00-dec-g}, ${base00-dec-b}" \
-          --replace "%BASE05%" "${base05-dec-r}, ${base05-dec-g}, ${base05-dec-b}"
-      ''
-    }
+    substituteInPlace $themeDir/stylix.script \
+      --replace "%BASE00%" "${base00-dec-r}, ${base00-dec-g}, ${base00-dec-b}" \
+      --replace "%BASE05%" "${base05-dec-r}, ${base05-dec-g}, ${base05-dec-b}"
```

Based on https://github.com/danth/stylix/pull/293 we should:

```suggestion
      --replace-fail "%BASE00%" "${base00-dec-r}, ${base00-dec-g}, ${base00-dec-b}" \
      --replace-fail "%BASE05%" "${base05-dec-r}, ${base05-dec-g}, ${base05-dec-b}"
```